Add `4 Digit World Cup 2025 Concludes` news post (#13546)
ppy/osu-wiki: news/2025/2025-07-20-4-digit-world-cup-2025-concludes.md

The antimod [FM2](https://osu.ppy.sh/beatmapsets/2388329#osu/5168342) pick was Canada's next choice. Both teams' No Mod players, ::{ flag=CA }:: [yip](https://osu.ppy.sh/users/5177569) and ::{ flag=PL }:: [szedis](https://osu.ppy.sh/users/14282987), put up solid scores in their own right, but the veteran ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) stole the show, holding exactly one thousand combo with HDHR for a score of 706,629 and securing his team the point. The follow-up [DT4](https://osu.ppy.sh/beatmapsets/2107364#osu/4494080) pick went in Poland's favor, with ::{ flag=PL }:: [szedis](https://osu.ppy.sh/users/14282987) and ::{ flag=PL }:: [Kosiarek](https://osu.ppy.sh/users/9086565) both getting solid scores of over 650,000 points.

> The final Free Mod pick for Canada in [FM3](https://osu.ppy.sh/beatmapsets/2388222#osu/5168188) went similarly to their prior picks. ::{ flag=CA }:: [Juan Dario](https://osu.ppy.sh/users/16529875) (a.k.a. Wanderio) and ::{ flag=CA }:: [Zucchiniii](https://osu.ppy.sh/users/13959253) secured the win with scores of 625,453 and 622,086 on No Mod and Hidden respectively. Poland, following in the footsteps of their earlier HR2 pick, ventured into [HR3](https://osu.ppy.sh/beatmapsets/2021678#osu/4261090). ::{ flag=PL }:: [Kosiarek](https://osu.ppy.sh/users/9086565) and ::{ flag=PL }:: [Asriel](https://osu.ppy.sh/users/11308471) both delivered 1 miss scores of 658,544 and 589,563, securing Poland match point at 6–4.

Reviewer: ```suggestion
The final Free Mod pick for Canada in [FM3](https://osu.ppy.sh/beatmapsets/2388222#osu/5168188) went similarly to their prior picks. ::{ flag=CA }:: [Juan Dario](https://osu.ppy.sh/users/16529875) (a.k.a. Wanderio) and ::{ flag=CA }:: [Zucchiniii](https://osu.ppy.sh/users/13959253) secured the win with scores of 625,453 and 622,086 on No Mod and Hidden respectively. Poland, following in the footsteps of their earlier HR2 pick, ventured into [HR3](https://osu.ppy.sh/beatmapsets/2021678#osu/4261090). ::{ flag=PL }:: [Kosiarek](https://osu.ppy.sh/users/9086565) and ::{ flag=PL }:: [Asriel](https://osu.ppy.sh/users/11308471) both delivered 1-miss scores of 658,544 and 589,563, securing Poland match point at 6–4.
```

It all came down to the tiebreaker. Poland and Canada were neck-and-neck for the first few minutes of the map. However, by the halfway point, Canada started to run away with it. ::{ flag=CA }:: [Juan Dario](https://osu.ppy.sh/users/16529875) and ::{ flag=CA }:: [Birchman](https://osu.ppy.sh/users/10676573) held crucial combos, countering a similar combo from ::{ flag=PL }:: [szedis](https://osu.ppy.sh/users/14282987) on the side of Poland. Once Canada had built their lead, it never went away, and they won the tiebreaker by 221,129 points, securing them the spot in the Grand Finals.

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/k4WLnqIMUdg" frameborder="0" allowfullscreen></iframe>
</div>

> Unfortunately, Canada's run would meet a swift end during their rematch against the United States. The Americans put their foot on the gas pedal and never let up, winning 7–0 in a dominating fashion. ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990) led the way with standout scores of 807,904 on [HD3](https://osu.ppy.sh/beatmapsets/2389014#osu/5169951) and 802,910 on [HD1](https://osu.ppy.sh/beatmapsets/2388276#osu/5168274), countering ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578)'s similarly impressive scores of 769,565 and 735,014 respectively. ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) also scored a ridiculous 722,439 with Hidden on [FM3](https://osu.ppy.sh/beatmapsets/2388222#osu/5168188), but unfortunately, the United States' overall team consistency, backed by ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) (a.k.a. -Schwarts) taking advantage of the EZ multiplier of 2.4 on the map to secure 609,091 points, was too much to overcome by himself.

Reviewer: usually written like this on osu!

```suggestion
Unfortunately, Canada's run would meet a swift end during their rematch against the United States. The Americans put their foot on the gas pedal and never let up, winning 7–0 in a dominating fashion. ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990) led the way with standout scores of 807,904 on [HD3](https://osu.ppy.sh/beatmapsets/2389014#osu/5169951) and 802,910 on [HD1](https://osu.ppy.sh/beatmapsets/2388276#osu/5168274), countering ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578)'s similarly impressive scores of 769,565 and 735,014 respectively. ::{ flag=CA }:: [Azer](https://osu.ppy.sh/users/2155578) also scored a ridiculous 722,439 with Hidden on [FM3](https://osu.ppy.sh/beatmapsets/2388222#osu/5168188), but unfortunately, the United States' overall team consistency, backed by ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) (a.k.a. -Schwarts) taking advantage of the EZ multiplier of 2.4x on the map to secure 609,091 points, was too much to overcome by himself.
```

## Match highlights

### Quarterfinals: ::{ flag=US }:: United States vs. ::{ flag=FR }:: France

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/9F8TnGamMKc" frameborder="0" allowfullscreen></iframe>
</div>

> What took place in the Quarterfinals weekend was perhaps the greatest clash of pure firepower in this iteration of 4 Digit World Cup: a match between the 9th seeded France and the 1st seeded United States.

Reviewer: ```suggestion
What took place in the Quarterfinals weekend was perhaps the greatest clash of pure firepower in this iteration of the 4 Digit World Cup: a match between the 9th-seeded France and the 1st-seeded United States.
```

> Going into this match, the average viewer likely expected this match to be a rout in favor of the two-time defending champion United States, whose roster had not lost a match since 2022. The first three maps of this matchup would support this outcome, with a decisive 662,297 victory on their first pick of [HD2](https://osu.ppy.sh/beatmapsets/2377013#osu/5135263), a 974,280 break-point on France's first pick of [DT4](https://osu.ppy.sh/beatmapsets/2377223#osu/5135786), and an overwhelming nearly 1.2 million score gap on their second pick of [NM6](https://osu.ppy.sh/beatmapsets/481703#osu/1430354). ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990) led the way for the Americans with FCs on HD2 and DT4, with the gimmick core of ::{ flag=US }:: [BATBALL](https://osu.ppy.sh/users/15173952), ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) (a.k.a. -Schwarts), and ::{ flag=US }:: [toybot](https://osu.ppy.sh/users/2848604) returning the favor on NM5. This tidal wave of American momentum seemed to be enough for them to coast to an easy win for the rest of the match, right? That would have been true in most matchups for them, but this French team was different. As their coach and 4WC commentator ::{ flag=FR }:: [Subaru_Arima](https://osu.ppy.sh/users/11273062) boldly claimed in the prediction stream prior to the tournament: they wouldn't go down so easily.

Reviewer: oxford comma isn't used earlier in the article
assuming the last part is a quote

```suggestion
Going into this match, the average viewer likely expected this match to be a rout in favour of the two-time defending champion United States, whose roster had not lost a match since 2022. The first three maps of this matchup would support this outcome, with a decisive 662,297 victory on their first pick of [HD2](https://osu.ppy.sh/beatmapsets/2377013#osu/5135263), a 974,280 break point on France's first pick of [DT4](https://osu.ppy.sh/beatmapsets/2377223#osu/5135786), and an overwhelming 1.2 million score gap on their second pick of [NM6](https://osu.ppy.sh/beatmapsets/481703#osu/1430354). ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990) led the way for the Americans with FCs on HD2 and DT4, with the gimmick core of ::{ flag=US }:: [BATBALL](https://osu.ppy.sh/users/15173952), ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) (a.k.a. -Schwarts) and ::{ flag=US }:: [toybot](https://osu.ppy.sh/users/2848604) returning the favour on NM5. This tidal wave of American momentum seemed to be enough for them to coast to an easy win for the rest of the match, right? That would have been true in most matchups for them, but this French team was different. As their coach and 4WC commentator ::{ flag=FR }:: [Subaru_Arima](https://osu.ppy.sh/users/11273062) boldly claimed in the prediction stream prior to the tournament, "they wouldn't go down so easily".
```

Author: would it be better to remove the comma here or add elsewhere

> The French counter-attack came in the form of alt and tech, sparked by a stunning 3-way FC and victory on their own [NM3](https://osu.ppy.sh/beatmapsets/2377170#osu/5135631) pick by ::{ flag=FR }:: [calangi](https://osu.ppy.sh/users/7810180), ::{ flag=FR }:: [fiaee](https://osu.ppy.sh/users/10325072), and ::{ flag=FR }:: [lonq noir](https://osu.ppy.sh/users/14165027). The United States followed by picking [HR3](https://osu.ppy.sh/beatmapsets/2377026#osu/5135285), which saw ::{ flag=FR }:: [lonq noir](https://osu.ppy.sh/users/14165027) carry a 262,791 victory for France, matching ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990)'s FC on the other side. France used their next pick on [NM4](https://osu.ppy.sh/beatmapsets/2042527#osu/4262164), in which ::{ flag=FR }:: [calangi](https://osu.ppy.sh/users/7810180) clocked a 5x100 FC and the rest of the French outscored three of the four of the Americans for their own 1.1 million score victory. Shockingly, the United States decided to use their fourth pick on [FM3](https://osu.ppy.sh/beatmapsets/2322063#osu/4975164): another alt map. Despite great efforts from ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) and ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990), France nearly repeated the scenario from NM3 with three scores above 800,000 being enough to hold off the Americans for another break point.

Reviewer: ```suggestion
The French counter-attack came in the form of alt and tech, sparked by a stunning 3-way FC and victory on their own [NM3](https://osu.ppy.sh/beatmapsets/2377170#osu/5135631) pick by ::{ flag=FR }:: [calangi](https://osu.ppy.sh/users/7810180), ::{ flag=FR }:: [fiaee](https://osu.ppy.sh/users/10325072) and ::{ flag=FR }:: [lonq noir](https://osu.ppy.sh/users/14165027). The United States followed by picking [HR3](https://osu.ppy.sh/beatmapsets/2377026#osu/5135285), which saw ::{ flag=FR }:: [lonq noir](https://osu.ppy.sh/users/14165027) carry a 262,791 victory for France, matching ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990)'s FC on the other side. France used their next pick on [NM4](https://osu.ppy.sh/beatmapsets/2042527#osu/4262164), in which ::{ flag=FR }:: [calangi](https://osu.ppy.sh/users/7810180) clocked a 5x100 FC along with teammates who outscored three of the four Americans for a 1.1 million score victory. Shockingly, the United States decided to use their fourth pick on [FM3](https://osu.ppy.sh/beatmapsets/2322063#osu/4975164), another alt map. Despite great efforts from ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) and ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990), France nearly repeated the scenario from NM3 with three scores above 800,000 being enough to hold off the Americans for another break point.
```

> In the blink of an eye, the momentum was now squarely on the side of France, who had rattled off 4 straight points and now had the opportunity to use their fourth pick to reach match point and potentially hand the United States their first loss in three years. They selected [HD3](https://osu.ppy.sh/beatmapsets/2373888#osu/5126280), which on paper was a great pick, being the final tech map in a matchup where they had won the other two. But the United States snapped right back, with a ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) FC and high supporting scores granting them a break point with a wide margin of 959,711. Now at 4–4, they would reach match point by picking the anti-mod [FM2](https://osu.ppy.sh/beatmapsets/2210416#osu/5132658) and dominating France with another nearly 1 million point difference.

Reviewer: ```suggestion
In the blink of an eye, the momentum was now squarely on the side of France, who had rattled off 4 straight points and now had the opportunity to use their fourth pick to reach match point and potentially hand the United States their first loss in three years. They selected [HD3](https://osu.ppy.sh/beatmapsets/2373888#osu/5126280), which on paper was a great pick, being the final tech map in a matchup where they had won the other two. The United States, however, snapped right back with a ::{ flag=US }:: [LUNAISTABBY](https://osu.ppy.sh/users/4673649) FC and high supporting scores granting them a break point with a wide margin of 959,711. Coming from a 4–4 tie, they then reached match point by picking the anti-mod [FM2](https://osu.ppy.sh/beatmapsets/2210416#osu/5132658) and dominating France with another nearly 1 million point difference.
```

> It was now France's final pick. With pressure at an all-time high, they decided to stake their winner's bracket lives on [HR2](https://osu.ppy.sh/beatmapsets/1262817#osu/2639476), a hybrid map of precision and flow aim alt. The map was close until the very end, with ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990) and ::{ flag=FR }:: [Draxarta](https://osu.ppy.sh/users/11129041) matching FCs, supported by ::{ flag=US }:: [Uzumaki](https://osu.ppy.sh/users/13310147) and ::{ flag=FR }:: [fiaee](https://osu.ppy.sh/users/10325072) matching chokes, but ultimately the supporting scores for the Americans were better as they finished off the French team on a nail-biter 78,969 victory for a final break point.

Reviewer: ```suggestion
It was now France's final pick. With pressure at an all-time high, they decided to stake their winners' bracket lives on [HR2](https://osu.ppy.sh/beatmapsets/1262817#osu/2639476), a hybrid map of precision and flow aim alt. The map was close until the very end, with ::{ flag=US }:: [rng_](https://osu.ppy.sh/users/9265990) and ::{ flag=FR }:: [Draxarta](https://osu.ppy.sh/users/11129041) matching FCs, supported by ::{ flag=US }:: [Uzumaki](https://osu.ppy.sh/users/13310147) and ::{ flag=FR }:: [fiaee](https://osu.ppy.sh/users/10325072) matching chokes, but ultimately the supporting scores for the Americans were better as they finished off the French team on a nail-biting 78,969-point victory.
```

> Both teams played incredibly well this match, with the United States team achieving the top score of the weekend on 5/6 of their points, and the French team doing so on 3/4 of theirs. Besides the final map, most picks in this match were won in dominant fashion, with only 2 other picks being decided by a margin of 450,000 or below, and many being close to 1 million; an extremely rare statistic for a match that finished at 6–4.

Reviewer: ```suggestion
Both teams played incredibly well this match, with the United States team achieving the top score of the weekend on 5/6 of their points, and the French team doing so on 3/4 of theirs. Besides the final map, most picks in this match were won in dominant fashion, with only 2 other picks being decided by a margin of 450,000 or below, and many being close to 1 million — an extremely rare statistic for a match that finished at 6–4.
```

### Semifinals: ::{ flag=DE }:: Germany vs. ::{ flag=ID }:: Indonesia

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/o3qqffDMWZg" frameborder="0" allowfullscreen></iframe>
</div>

> Moving into the Semifinals weekend, the 2nd seed Germany, after their unexpected loss against Sweden in the previous week, had to face off against the 17th seed Indonesian roster to avoid being eliminated.

Reviewer: ```suggestion
Moving into the Semifinals weekend, the 2nd-seeded Germany, after their unexpected loss against Sweden in the previous week, had to face off against the 17th-seeded Indonesian roster to avoid being eliminated.
```

Indonesia, the dark horse of this match, started off strong by stealing Germany's [DT1](https://osu.ppy.sh/beatmapsets/2380996#osu/5150032) pick, where ::{ flag=ID }:: [Rin Kokonoe](https://osu.ppy.sh/users/11091594) (a.k.a. ThatNOOBguy) managed to outlast ::{ flag=DE }:: [kittenfrost](https://osu.ppy.sh/users/11234706) (a.k.a. ZeitFrost) with a 1,126,633 FC, resulting in the first break point of the match. Germany responded back with equal power, taking a break point of their own on Indonesia's [NM3](https://osu.ppy.sh/beatmapsets/2380835#osu/5146682) pick, tying the game 1–1.

> Germany's next pick of [FM3](https://osu.ppy.sh/beatmapsets/2226198#osu/4722524), one of the hardest maps in the pool, attempted to capitalise on the skillcap advantage over Indonesia. However, ::{ flag=ID }:: [salt and honey](https://osu.ppy.sh/users/8688737) had another plan in mind, pulling out a crucial 971,689 Hidden FC to secure Indonesia another break point. This enabled the Indonesians to finally consolidate and end the streak of break points on [FM1](https://osu.ppy.sh/beatmapsets/2040149#osu/4255853), the awkward aim map of the pool, thanks to ::{ flag=ID }:: [FAW](https://osu.ppy.sh/users/11070577)'s 870,256 near-FC with Easy. Powered by the momentum, Indonesia took yet another break point on [NM4](https://osu.ppy.sh/beatmapsets/1851103#osu/3802734), putting Germany into an insanely dangerous scoreline of 1–4, without having won any of their own picks so far.

Reviewer: ```suggestion
Germany's next pick of [FM3](https://osu.ppy.sh/beatmapsets/2226198#osu/4722524), one of the hardest maps in the pool, attempted to capitalise on the skill cap advantage over Indonesia. However, ::{ flag=ID }:: [salt and honey](https://osu.ppy.sh/users/8688737) had another plan in mind, pulling out a crucial 971,689 Hidden FC to secure Indonesia another break point. This enabled the Indonesians to finally consolidate and end the streak of break points on [FM1](https://osu.ppy.sh/beatmapsets/2040149#osu/4255853), the awkward aim map of the pool, thanks to ::{ flag=ID }:: [FAW](https://osu.ppy.sh/users/11070577)'s 870,256 near-FC with Easy. Powered by the momentum, Indonesia took yet another break point on [NM4](https://osu.ppy.sh/beatmapsets/1851103#osu/3802734), putting Germany into an insanely dangerous scoreline of 1–4, without having won any of their own picks so far.
```

After a few more traded break points on [HD1](https://osu.ppy.sh/beatmapsets/1674871#osu/3421429) and [DT3](https://osu.ppy.sh/beatmapsets/1945323#osu/4144405) and with a scoreline of 2–5, Germany was moments away from facing elimination. Miraculously, with a point difference of just 14,116, they delayed this fate by taking Indonesia's [NM2](https://osu.ppy.sh/beatmapsets/375956#osu/823257) pick despite a shaky ending, resulting in the 7th break point of the match. A subsequent win on [DT4](https://osu.ppy.sh/beatmapsets/1853565#osu/3808761) placed the scoreline at 4–5, with Indonesia having the choice for the last pick of the match.

> That last pick, and the last chance for Germany to force the tiebreaker, ended up being [HR2](https://osu.ppy.sh/beatmapsets/2381120#osu/5147502), the precision pick. Despite best efforts and a stellar 856,063 reverse choke from ::{ flag=DE }:: [stellasu](https://osu.ppy.sh/users/8508931), Germany failed to pull back the score by just 8,686 points. As such, Indonesia had eliminated the 2nd seed powerhouse of Germany from 4WC, finishing the biggest upset of the tournament.

Reviewer: ```suggestion
That last pick, and the last chance for Germany to force the tiebreaker, ended up being [HR2](https://osu.ppy.sh/beatmapsets/2381120#osu/5147502), the precision pick. Despite best efforts and a stellar 856,063 reverse choke from ::{ flag=DE }:: [stellasu](https://osu.ppy.sh/users/8508931), Germany failed to pull back the score by just 8,686 points. As such, Indonesia had eliminated the 2nd-seed powerhouse of Germany from 4WC, finishing the biggest upset of the tournament.
```
